website/src/components/DataTable/DataTable.tsx

```diff
@@ -199,7 +199,7 @@ const DataTableRow = <T,>({ row }: { row: Row<T> }) => {
       {renderCells.map((cell) => {
         const props = cell.column.columnDef.meta?.cellProps?.(cell);
         return (
-          <Td key={cell.id} {...props} colSpan={cell.span}>
+          <Td key={cell.id} {...props} colSpan={cell.span} style={{ overflow: "hidden" }}>
```

Comment:
We dont want to add this bahavior on the DataTable component

Reply:
I don't entirely see why not, nor how you'd go about fixing this problem in places like the Users list which used data tables. Am I missing something, how else should I go about doing this?

Reply:
@scorbett123 
notmd is pointing out that you shouldn't put this use-case specific style in the general 'DataTable' component.

Either have a TdStyle property that you pass through, or simply set css property where you _use_ DataTable, like:

.mydatatable td { overflow: hidden }

Reply:
There is a cellProps prop. You can use it to pass the style.

Reply:
Thanks, is this what you're meaning?